### DISCUSSION TITLE
aux_load1_extra and aux_load2_extra values

### DISCUSSION THREAD

**Discussion by stedi73** [slipx06/sunsynk-power-flow-card#366] [General]
hi, I think it would be good idea when the AUX is in "off mode"  the aux_load1_extra and aux_load2_extra  values are visible ( for example pool temperature without the pump on)

**Comment by slipx06** [comment]
Hi. Those values are never hidden. There was an issue with aux_load1_extra which is fixed in the last release. v4.28.1

**Reply by stedi73** [reply]
Thanks
